src/clusterfuzz/_internal/crash_analysis/crash_analyzer.py

```diff
@@ -101,6 +101,7 @@
     # math of crpto libraries.
     'Unsigned-integer-overflow',
     'Unreachable code',
+    'Implicit-conversion',
```

Comment:
Do we need to maintain an alphabetical order of this list?

Reply:
Yes good catch! let's maintain it to enable easier searching by humans.